### ISSUE TITLE
HTML Validator Accessibility errors - div not allowed in span

### ISSUE THREAD

**Issue by espeo196** [OfficeDev/office-ui-fabric-react#11431]
### Describe the issue:

When running the [HTML Validator](https://validator.w3.org/nu/about.html) I came across a few issues similar to what was mentioned in [this issue](https://github.com/OfficeDev/office-ui-fabric-react/issues/10021) regarding invalid element nesting.

Of note:

- **Persona** - Element div not allowed as a child of element span.
    - `<div class="ms-Persona-coin ms-Persona--size32 coin-343">`
- **Toggle** - Element div not allowed as child of element button in this context
    - `<div class="ms-Toggle-thumb thumb-524">`

### Reproduce

You can go to the [toggle](https://developer.microsoft.com/en-us/fabric#/controls/web/toggle) and [persona](https://developer.microsoft.com/en-us/fabric#/controls/web/persona) pages and validate using the [Html validator](https://validator.w3.org/nu/about.html).

**Comment by micahgodbolt** [comment]
yeah, we did this fix in the past with buttons and other controls. These should be simple enough

**Comment by davallen** [comment]
Thanks Micah! Will track this thread for updates towards resolution.

**Comment by micahgodbolt** [comment]
@davallen little confused as to the persona issue. Where is the span that's containing a div? The entire persona only has 1 span, and it only is ever passed an initial string.

**Comment by davallen** [comment]
@micahgodbolt this is an interesting case. Looks like we put the coin inside a button. Is the button the issue then?

<button type="button" role="menuitem" name="Profile and Settings" value="Profile and Settings" class="ms-Button ms-Button--commandBar ms-Button--hasMenu ms-CommandBarItem-link root-277" aria-label="Profile and Settings" data-is-focusable="true" aria-expanded="false" aria-haspopup="true" tabindex="-1"><span class="ms-Button-flexContainer flexContainer-266" data-automationid="splitbuttonprimary"><div class="ms-Persona-coin ms-Persona--size32 coin-282"><div class="ms-Persona-imageArea imageArea-284"><div class="ms-Persona-initials initials-287" aria-hidden="true"><span>NJ</span></div></div></div></span></button>

**Comment by micahgodbolt** [comment]
you can use three backticks to make a code block
  ` ``` `
  ```code here```
` ``` `

**Comment by davallen** [comment]
@micahgodbolt - thank you! updated the code snippet

**Comment by micahgodbolt** [comment]
is the HTML validator error something stopping you from shipping your code? or is it a nice to have fix?

One solution is to just use a div with onClick event on it. That'll avoid using button with div in it, though you need to ensure that the div is accessible.

**Comment by davallen** [comment]
So this is an interesting component. It is a CommandBar with a MenuButton that uses the Persona coin as the icon. The menubutton then has a number of submenus.

It isn't blocking shipping, but it is getting flagged as an accessibility issue and stopping us from getting compliant.

**Comment by micahgodbolt** [comment]
yeah, it's tricky. Persona is a stand alone control. making all the divs flippable to spans is a slipperly slope. Do we need to do that with all of our controls?

**Comment by micahgodbolt** [comment]
I think limiting the scope to the Coin isn't bad though. Let me look at that.
